src/wp-includes/theme.php

```diff
@@ -4416,3 +4445,45 @@ static function ( $active, WP_Customize_Panel $panel ) {
 		2
 	);
 }
+
+/**
+ * Loads view transitions for the current theme, if configured.
+ *
+ * @since 6.9.0
+ */
+function wp_load_view_transitions() {
+	if ( ! current_theme_supports( 'view-transitions' ) ) {
+		return;
+	}
+
+	$stylesheet = file_get_contents( ABSPATH . WPINC . '/view-transitions.css' );
```

Comment:
When this file is minified we should use it instead: 

```suggestion
	$stylesheet = file_get_contents( ABSPATH . WPINC . '/view-transitions' . ( SCRIPT_DEBUG ? '.css' : '.min.css' ) );
```

This depends on moving the CSS file to `wp-includes/css`.

Reply:
Yes, in principle both the JS and CSS file should live somewhere else. Since this is just an experimental PR at this point, I didn't want to bother thinking about it (could even become a Gutenberg-first implementation and then be a new JS package, for example).

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/8370/commits/816315910a9147899a5cff75cdcdb092dadffeb8